sdk/identity/identity/src/credentials/managedIdentityCredential/index.ts

```diff
@@ -33,6 +33,18 @@ export interface ManagedIdentityCredentialResourceIdOptions extends TokenCredent
   resourceId: string;
 }
 
+/**
```

Comment:
@scottaddie any feedback on the doc-comments would be appreciated. Please review the versions above where we define `ManagedIdentityCredentialResourceIdOptions` and `ManagedIdentityCredentialClientIdOptions` - I tried to stay consistent with their verbiage but happy to change any / all doc-comments